### PR TITLE
Replaced digit 1 with 2 on second_branch first_file.txt

### PR DESCRIPTION
**Pull request by ArtiomLevitchii**
Merging with master branch

### CODE REVIEW

**Comment by ArtiomLevitchii** [comment]
Waiting for review

File: first_file.txt [hunk 1]
```diff
@@ -1 +1 @@
-1
\ No newline at end of file
+3
```
Comment (gh3rtzz) [inline first_file.txt]: Good job. Now change the number 3 to number 4

**Comment by ArtiomLevitchii** [comment]
Waiting to merge with Master after Hotfix

**Review by gh3rtzz — APPROVED** [review]
Approved. Go ahead and merge it

**Comment by ArtiomLevitchii** [comment]
Done

**Pull request merged**